pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -158,7 +158,8 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
         }
 
         // Initialize inverted index creator
-        if (invertedIndexColumns.contains(columnName)) {
+        // Skip creating inverted index if sorted; _invertedIndexCreatorMap is also correctly populated for metadata writing
```

Comment:
I don't quite follow the second part of the comment. "Skip creating inverted index if sorted" is good enough IMO.

Reply:
Standalone test was performed with pinot-admin commands. No inverted index was generated for sorted columns when I asked the system to do so with generatorConfigFile option.

Reply:
Can you pls merge it? Thx.